cudax/include/cuda/experimental/__execution/policy.cuh

```diff
@@ -57,6 +60,46 @@ _CCCL_INLINE_VAR constexpr bool __is_unsequenced_execution_policy =
   _Policy == execution_policy::unsequenced_host || _Policy == execution_policy::unsequenced_device
   || _Policy == execution_policy::parallel_unsequenced_host || _Policy == execution_policy::parallel_unsequenced_device;
 
+template <class _Tp>
+_LIBCUDACXX_CONCEPT_FRAGMENT(__has_member_get_execution_policy_,
+                             requires(const _Tp& __t)(requires(_CCCL_TRAIT(
+                               _CUDA_VSTD::is_convertible, decltype(__t.get_execution_policy()), execution_policy))));
+
+struct get_execution_policy_t;
+
+template <class _Tp>
+_LIBCUDACXX_CONCEPT __has_member_get_execution_policy = _LIBCUDACXX_FRAGMENT(__has_member_get_execution_policy_, _Tp);
+
+template <class _Env>
+_LIBCUDACXX_CONCEPT_FRAGMENT(
+  __has_query_get_execution_policy_,
+  requires(const _Env& __env, const get_execution_policy_t& __cpo)(
+    requires(!__has_member_get_execution_policy<_Env>),
+    requires(_CCCL_TRAIT(_CUDA_VSTD::is_convertible, decltype(__env.query(__cpo)), execution_policy))));
+
+template <class _Env>
+_LIBCUDACXX_CONCEPT __has_query_get_execution_policy = _LIBCUDACXX_FRAGMENT(__has_query_get_execution_policy_, _Env);
```

Comment:
death to `CONCEPT_FRAGMENT`

```c++
template <class _Env>
_LIBCUDACXX_CONCEPT __has_query_get_execution_policy =
  _LIBCUDACXX_REQUIRES_EXPR((_Env), const _Env& __env)(
    requires(!__has_member_get_execution_policy<_Env>),
    requires(_CUDA_VSTD::convertible_to<__query_result_t<get_execution_policy_t, _Env>,
                                        execution_policy>));

Reply:
Oh that might work, I found that `_LIBCUDACXX_REQUIRES_EXPR` does not work well when putting e.g `decltype(__env.query(__cpo))` in it

Reply:
Seems something else was fishy, fixed